docs/src/pages/guides/publish-to-npm.md

```diff
@@ -1,9 +1,10 @@
 ---
 layout: ~/layouts/MainLayout.astro
 title: Publish to NPM
+modifiedDate: '2021-09-06 @ 11:28:21'
 ---
 
-Building a new Astro component? **Publish it to [npm!](https://npmjs.com/)**
+Building a new Astro component? **Publish it to [npm!][NPM]**
```

Comment:
I actually personally prefer inline URLs, but I have started to see how this makes future refactoring much better, which is a pretty nice reason to use them. I'll swallow my personal pref and go +1 for doing this here and across the site moving forward

Reply:
I wasn't sure where you leaned so I'm glad it worked out for the best ^_^